docs/reference/adobe/experience/decisioning/scope-details.schema.md

```diff
@@ -81,6 +82,30 @@ The activity that was a requested or chosen for this decision scope. Some scopes
 
 
 
+## xdm:characteristics
+### Scope Details Characteristics
```

Comment:
Should we rephrase this to `Scope Characteristics`

Reply:
ok, thank you